src/hive/state.cljs

```diff
@@ -1,7 +1,8 @@
 (ns hive.state
   (:require [hive.rework.core :as work]
             [hive.rework.util :as tool]
-            [cljs.spec.alpha :as s]))
+            [cljs.spec.alpha :as s]
+            [figwheel.client.utils :as utils]))
```

Comment:
@mehdisadeghi please dont include figwheel into the app itself. 

figwheel is only for code reloading on development, never meant to be included as part of a production app. 

I think you mentioned it but what would be the problem with just using `js/console.error`? I thought about it and to avoid having weird stacktraces I think that we should always return `m` but only print the error if it is not valid.

React native should show the red screen whenever console.error is called, see: https://facebook.github.io/react-native/docs/debugging.html#errors